test/readme.md

```diff
@@ -241,6 +241,52 @@ beforeEach(() => {
 });
 ```
 
+## Running Benchmarks
+
+```sh
+npm run check:bench
+```
+
+Refer to the `run-spec-benchmark-tests-node-server` task for Node.js version, MongoDB server version, and platform that we run benchmarks against in CI.
+
+The server is run in standalone mode and the server versions are aliased by this script: https://github.com/mongodb-labs/drivers-evergreen-tools/blob/5048cca80e9ca62642409de2d401058bbd7057fa/.evergreen/mongodl.py#L58 check the latest version to see what alias the driver is running against.
+
+The host used is described here: https://spruce.mongodb.com/distro/rhel90-dbx-perf-large/settings/general (Auth required to view)
```

Comment:
Do we want to just describe it explicitly instead of linking to it? If external users or contributors want to do testing of their own, it may be useful for them to have an idea of what specs we're working with, especially if they want to do the bulk of their own performance work and present us with changes with perf improvements, it would cut down on PR review cycles if they had those details to spin up their own perf testing setups.

Reply:
added, although I don't expect users to reproduce the host, but I agree might as well document it.